libcontainer/container.go

```diff
@@ -30,6 +30,23 @@ const (
 	Destroyed
 )
 
+func (s Status) String() string {
+	switch s {
+	case Running:
+		return "running"
+	case Pausing:
+		return "pausing"
+	case Paused:
+		return "paused"
+	case Checkpointed:
```

Comment:
I'd rather not remove the const just yet until I finish the state refactoring with the new spec types to make sure this state will not be used anymore.